mex/model/entities/bibliographic-resource.json

```diff
@@ -0,0 +1,416 @@
+                },
+                {
+                    "type": "null"
+                }
+            ],
+            "default": null,
+            "description": "Date of formal issuance of the publication ([DCT, 2020-01-20](http://dublincore.org/specifications/dublin-core/dcmi-terms/2020-01-20/)).",
+            "sameAs": [
+                "http://purl.org/dc/terms/created"
+            ]
+        },
+        "journal": {
+            "default": [],
+            "description": "The periodical in which the article was published.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "type": "array"
+        },
+        "keyword": {
+            "default": [],
+            "description": "A keyword or tag describing the resource ([DCAT, 2020-02-04](https://www.w3.org/TR/2020/REC-vocab-dcat-2-20200204/)).",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "sameAs": [
+                "http://www.w3.org/ns/dcat#keyword"
+            ],
+            "type": "array"
+        },
+        "language": {
+            "default": [],
+            "description": "The language in which the publication was written.",
+            "items": {
+                "$ref": "/schema/entities/concept#/identifier",
+                "examples": [
+                    "https://mex.rki.de/item/language-1"
+                ],
+                "useScheme": "https://mex.rki.de/item/language"
+            },
+            "sameAs": [
+                "http://purl.org/dc/terms/language"
+            ],
+            "type": "array"
+        },
+        "license": {
+            "anyOf": [
+                {
+                    "$ref": "/schema/entities/concept#/identifier",
+                    "examples": [
+                        "https://mex.rki.de/item/license-1"
+                    ],
+                    "useScheme": "https://mex.rki.de/item/license"
+                },
+                {
+                    "type": "null"
+                }
+            ],
+            "default": null,
+            "description": "A legal document giving official permission to do something with the publication ([DCT, 2020-01-20](http://dublincore.org/specifications/dublin-core/dcmi-terms/2020-01-20/)).",
+            "sameAs": [
+                "http://purl.org/dc/terms/license"
+            ]
+        },
+        "pages": {
+            "description": "The range of pages or a single page.",
+            "examples": [
+                "1",
+                "45-67",
+                "45 - 67",
+                "II",
+                "XI",
+                "10i"
+            ],
+            "type": "string"
+        },
+        "publicationYear": {
+            "describing": "The year in which the publication was issued.",
+            "examples": [
+                "2011",
+                "1982",
+                "1810"
+            ],
+            "pattern": "^(?:\\d{4})$",
```

Comment:
this introduces a new timestamp format. up unitl now, we only had `YearMonthDayTime`, `YearMonthDay` and `YearMonth`. the latter has a required year and an optional month, but we cannot enforce that _just_  a year is given.

tldr; we have to convert each string with pattern `^(?:\\d{4}|(?:\\d{4}-(?:0[1-9]|1[0-2])))$` into two separate entries within their respective `anyOf`-lists. one for `YearMonth` (both required) and one for just `Year`

Reply:
okay, I wasn't aware of that, sorry. for publications, it is common to only give the year, so we cannot change that. but your suggestion sound good to me! I will change it accordingly.

Reply:
like that?
`        "publicationYear": {
            "anyOf": [
                {
                    "examples": [
                        "2011",
                        "1982",
                        "1810"
                    ],
                    "pattern": "^(?:\\d{4})$",
                    "type": "string"
                },
                {
                    "examples": [
                        "1903-01",
                        "2019-03"
                    ],
                    "pattern": "^(?:\\d{4}|(?:\\d{4}-(?:0[1-9]|1[0-2])))$",
                    "type": "string"
                }
            ],
            "description": "The year in which the publication was issued.",
            "sameAs": "http://datacite.org/schema/kernel-4/publicationYear",
            "type": "array"
        },`

> convert each string with pattern ^(?:\\d{4}|(?:\\d{4}-(?:0[1-9]|1[0-2])))$
do you mean for all properties in all entities?

Reply:
```suggestion
                    "publicationYear": {
            "anyOf": [
                {
                    "examples": [
                        "2011",
                        "1982",
                        "1810"
                    ],
                    "pattern": "^(?:\\d{4})$",
                    "type": "string"
                },
                {
                    "examples": [
                        "1903-01",
                        "2019-03"
                    ],
                    "pattern": "^(?:\\d{4}|(?:\\d{4}-(?:0[1-9]|1[0-2])))$",
                    "type": "string"
                }
            ],
            "description": "The year in which the publication was issued.",
            "sameAs": "http://datacite.org/schema/kernel-4/publicationYear",
            "type": "array"
        },
```

Reply:
like that?

`        "publicationYear": {
            "anyOf": [
                {
                    "examples": [
                        "2011",
                        "1982",
                        "1810"
                    ],
                    "pattern": "^(?:\\d{4})$",
                    "type": "string"
                },
                {
                    "examples": [
                        "1903-01",
                        "2019-03"
                    ],
                    "pattern": "^(?:\\d{4}|(?:\\d{4}-(?:0[1-9]|1[0-2])))$",
                    "type": "string"
                }
            ],
            "description": "The year in which the publication was issued.",
            "sameAs": "http://datacite.org/schema/kernel-4/publicationYear",
            "type": "array"
        },
`

Reply:
can you please take a look, if I applied the changes correctly in `publicationYear` ?

Reply:
> do you mean for all properties in all entities?

yes, would be more consistent

Reply:
actually, our patterns were a bit inconsistent. i fixed them:
```json
{
    "year_month_day_time": "^\\d{4}-(?:0[1-9]|1[0-2])-(?:0[1-9]|[12][0-9]|3[01])T(?:[0-1][0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9]Z$",
    "year_month_day": "^\\d{4}-(?:0[1-9]|1[0-2])-(?:0[1-9]|[12][0-9]|3[01])$",
    "year_month": "\\d{4}-(?:0[1-9]|1[0-2])$",
    "year": "^\\d{4}$"
}
```